File: clients/java/examples/admin-find-database.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;
import com.datastax.astra.client.databases.definition.DatabaseInfo;
import java.util.UUID;

public class Example {
  public static void main(String[] args) {
    DataAPIClient client = new DataAPIClient("**APPLICATION_TOKEN**");

    AstraDBAdmin admin = client.getAdmin();

    DatabaseInfo databaseInfo = admin.getDatabaseInfo(UUID.fromString("**DATABASE_ID**"));

    System.out.println(databaseInfo);
  }
}

File: clients/csharp/examples/admin-find-database.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    var client = new DataAPIClient("**APPLICATION_TOKEN**");

    var admin = client.GetAstraDatabasesAdmin();

    var info = await admin.GetDatabaseInfoAsync("**DATABASE_ID**");
  }
}
